### PR TITLE
Final website code

### PR DESCRIPTION
**Pull request by Thejus5**
Final code of the responsive layout. Responsive at max-widths 768px and 416px (Tab and mobile view).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+    </section>
+    <!-- Project section ends here -->
+
+    <!-- Testimonials starts here -->
+    <section class="testimonials container">
+      <div class="smallHeading centeredText">Testimonials</div>
+      <h3 class="centeredText">Clients Say About Us</h3>
+      <div class="flexBox testimonialBox">
+        <div class="testimonialsImage">
+          <img src="Resources/images/testimonial/testimonial-1.jpg" alt="Smiling Man">
```
Comment (karthikamg) [inline index.html]: Provide meaningful names to images.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+    </section>
+    <!-- Services section ends here -->
+
+    <!-- Projects section starts here -->
+    <section class="project container">
+      <div class="smallHeading centeredText">Works We Done</div>
+      <h3 class="centeredText">Our Recent Projects</h3>
+      <div class="projectsImageBoxes flexBox">
+        <div class="projectImageBox">
+          <img src="Resources/images/projects/img-1.jpg" alt="">
```
Comment (karthikamg) [inline index.html]: Provide meaningful names to images throughout the site.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+              literature
+              from 45 BC</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- Offer section ends here -->
+
+    <!-- About section starts here -->
+    <section class="about container flexBox" id="about">
```
Comment (karthikamg) [inline index.html:82]: Code and style from first section can be reused here.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+      <div class="usefulLinks">
+        <h4>Useful Links</h4>
+        <ul>
+          <li><a href="#">About Us</a></li>
+          <li><a href="#">Services</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </div>
+
+      <div class="contact1">
```
Comment (karthikamg) [inline index.html]: Don't provide class names as 1, 2 etc. Provide meaningful names.

File: Resources/css/style.css [hunk 5]
```diff
@@ -0,0 +1,433 @@
...
+.bannerTextBox a{ margin-top: 20px; }
+
+.bannerTextBox span{ color: #709d61;}
+
+.bannerTextBox{ padding-top: 90px; }
+
+.bannerImageBox{ padding: 20px;}
+
+.bannerImageBox img{
+  width: 600px;
```
Comment (karthikamg) [inline Resources/css/style.css:187]: Always try not to give hard coded sizes to images unless and until there is no other way, as this affect image quality. Better provide image as background.

File: Resources/css/style.css [hunk 6]
```diff
@@ -0,0 +1,433 @@
...
+}
+
+.btn-ghost{
+  border: 1px solid #709d61;
+  color: #709d61;
+  -webkit-transition: all 0.2s ease;
+  transition: all 0.2s ease;
+}
+
+.btn-full:hover{
```
Comment (karthikamg) [inline Resources/css/style.css:129]: Button hover effects must be consistent through out the site.

File: Resources/css/style.css [hunk 7]
```diff
@@ -0,0 +1,433 @@
...
+  outline: none;
+  padding-left: 10px;
+}
+input[type='button']{
+  font-size: 125%;
+  outline: none;
+}
+
+::-webkit-input-placeholder{
+  font-size: 112.5%;
```
Comment (karthikamg) [inline Resources/css/style.css]: Always try to provide font size in rem or em or otherwise px.

File: Resources/css/style.css [hunk 8]
```diff
@@ -0,0 +1,433 @@
...
+/* -------------------------------- */
+/* 6. About */
+/* -------------------------------- */
+
+.aboutTextBox{ padding: 10px;}
+
+.aboutTextBox a{ margin-top: 35px;}
+
+.aboutTextBox h3{
+  font-size: 281%;
```
Comment (karthikamg) [inline Resources/css/style.css]: Better to provide font size in re, or em or otherwise in px in all occurence.

File: Resources/css/style.css [hunk 9]
```diff
@@ -0,0 +1,433 @@
...
+/* -------------------------------- */
+/* 11. Footer */
+/* -------------------------------- */
+hr{ opacity: 0.2;}
+
+.links{ padding-bottom: 0;}
+
+.gardenerLogo img{padding: 15px 0; }
+
+.links div{
```
Comment (karthikamg) [inline Resources/css/style.css:414]: Always try to avoid direct styling to generic elements.

File: index.html (hunk 1 above)
Comment (Thejus5) [reply index.html]: The names has been changed to meaningful ones.

File: index.html (hunk 3 above)
Comment (Thejus5) [reply index.html:82]: Some properties that does not require alterations has been reused.

File: index.html (hunk 2 above)
Comment (Thejus5) [reply index.html]: Meaningful names are given.

File: index.html (hunk 4 above)
Comment (Thejus5) [reply index.html]: Changed the names to meaningful text

File: Resources/css/style.css (hunk 5 above)
Comment (Thejus5) [reply Resources/css/style.css:187]: Replaced instances with classnames

File: Resources/css/style.css (hunk 6 above)
Comment (Thejus5) [reply Resources/css/style.css:129]: Consistent hover effect added.

File: Resources/css/style.css (hunk 7 above)
Comment (Thejus5) [reply Resources/css/style.css]: Font sizes replaced with rem

File: Resources/css/style.css (hunk 8 above)
Comment (Thejus5) [reply Resources/css/style.css]: Font sizes replaced with rem

**Pull request merged**